src/components/UpdatingForm/UpdatingForm.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
+
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  title: string,
+  id: number,
+  deleteTodo: (todoId: number) => void,
+  updateTodo: (
+    todoId: number,
+    newTodoData: Partial<Pick<Todo, 'title' | 'completed'>>
+  ) => void,
```

Comment:
Same as in `todoinfo` component
Create a handler for this function in your app component.

```suggestion
  onUpdateTodo: (todo: Todo) => void,
```